docs/cli-reference.md

```diff
@@ -0,0 +1,572 @@
+# CLI Reference
+
+## Overview
+
+The Relayer SDK (v0.4.0-alpha.2) includes command-line tools for testing, debugging, and managing FHEVM operations. These tools help developers verify their setup, test cryptographic operations, and debug integration issues without writing code.
+
+---
+
+## Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Configuration Commands](#configuration-commands)
+- [Cryptographic Operations](#cryptographic-operations)
+- [Public Key Management](#public-key-management)
+- [Testing Utilities](#testing-utilities)
+
+---
+
+## Installation
+
+The CLI tools are included with the SDK:
+
+```bash
+npm install @zama-ai/relayer-sdk@0.4.0-alpha.2
```

Comment:
Small typo, it should be `@zama-fhe/relayer-sdk@0.4.0-alpha.2` not `@zama-ai/relayer-sdk@0.4.0-alpha.2`